.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,262 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) issue comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  parse_comment:
+    if: startsWith(github.event.comment.body, 'Github, generate report for')
+    runs-on: ubuntu-latest
+    outputs:
+      check_name: ${{ steps.parse.outputs.check_name }}
+      example: ${{ steps.parse.outputs.example }}
+      config_link: ${{ steps.fetch_readme.outputs.config_link }}
+      projects_link: ${{ steps.fetch_readme.outputs.projects_link }}
+      branch: ${{ steps.branch.outputs.ref }}
+      commit_sha: ${{ steps.branch.outputs.commit_sha }}
+
+    steps:
+      - uses: shanegenschaw/pull-request-comment-trigger@v2.1.0
+        name: React with rocket on run
+        with:
+          trigger: ','
+          reaction: rocket
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+      - run: 'echo We print it here for this action to work'
+        if: 'true'
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Parse comment
+        id: parse
+        run: |
+          COMMENT="${{ github.event.comment.body }}"
+          CHECK_EXAMPLE=$(echo "$COMMENT" | grep -oP '(?<=Github, generate report for ).*')
+          CHECK_NAME=$(echo "$CHECK_EXAMPLE" | cut -d'/' -f1)
+          EXAMPLE=$(echo "$CHECK_EXAMPLE" | cut -d'/' -f2)
+          echo "CHECK_NAME=$CHECK_NAME"
+          echo "EXAMPLE=$EXAMPLE"
+          ./.ci/append-to-github-output.sh "check_name" "$CHECK_NAME"
+          ./.ci/append-to-github-output.sh "example" "$EXAMPLE"
+
+      - name: Fetch README
+        id: fetch_readme
+        run: |
+          README_URL="https://raw.githubusercontent.com/checkstyle/test-configs/main/${{ steps.parse.outputs.check_name }}/${{ steps.parse.outputs.example }}/README.md"
```

Comment:
Instead of readme, take config.xml directly.

Reply:
Done